exporters/elasticsearch/include/opentelemetry/exporters/elasticsearch/es_log_exporter.h

```diff
@@ -49,12 +53,19 @@ struct ElasticsearchExporterOptions
                                int port             = 9200,
                                std::string index    = "logs",
                                int response_timeout = 30,
-                               bool console_debug   = false)
-      : host_{host},
-        port_{port},
-        index_{index},
-        response_timeout_{response_timeout},
-        console_debug_{console_debug}
+                               bool console_debug   = false
+#  ifdef ENABLE_ASYNC_EXPORT
+                               ,
+                               bool asynchronous_mode = false
+#  endif
+                               )
+      : host_{host}, port_{port}, index_{index}, response_timeout_{response_timeout}, console_debug_
+  {
+    console_debug
+  }
+#  ifdef ENABLE_ASYNC_EXPORT
+  , asynchronous_mode_ { asynchronous_mode }
+#  endif
```

Comment:
Just a thought - should we still keep sync mode, or can we have async as default and only option?

Reply:
Sure, OTLP may also need always return `ExportResult::kSuccess` and ignore the result of HTTP client